_about/about.md

```diff
@@ -4,6 +4,8 @@ title: What is Asylo?
 
 overview: Learn why Asylo could be right for your application.
 
+keywords: about, about asylo, what is asylo?, asylo, info, about enclave application.
```

Comment:
To keep this review discussion manageable, please refrain from customizing individual page keywords in this PR. Just adding the ability for per-page customization and updating the default list.

Reply:
so do I have to remove the keywords from other pages? or I just comment them?

Reply:
Yes that is what we need to do. This PR is mainly geared towards adding just the functionality of page specific keyword and update default keywords.

Reply:
reverted some changes, please check